packages/protocol/tests/generate-test-vectors.ts

```diff
@@ -104,6 +104,55 @@ const generateParseRfqVector = async () => {
   }
 }
 
+const generateParseRfqOmitPrivateDataVector = async () => {
+  const vc = await VerifiableCredential.create({
+    type    : 'PuupuuCredential',
```

Comment:
is it 💩 💩? or is it 🔫 🔫 ? what's the difference?

```suggestion
    type    : 'PewPewCredential',
```